tests/z_msgcodec_test.c

```diff
@@ -1339,7 +1352,16 @@ _z_zenoh_message_t gen_query_message(void) {
     z_consolidation_mode_t consolidation;
     consolidation = con[gen_uint8() % (sizeof(con) / sizeof(uint8_t))];
 
-    return _z_msg_make_query(key, parameters, qid, target, consolidation);
+    _z_value_t with_value;
+    if (gen_bool()) {
+        with_value = gen_with_value();
+    } else {
+        with_value.encoding.prefix = Z_ENCODING_PREFIX_EMPTY;
```

Comment:
Should have a constructor for the empty value?

Reply:
As of now, I think it would just be useful for the tests.
It might be implemented in the future, if we see the need for it.